apps/schematic/api/schematic_api/openapi/openapi.yaml

```diff
@@ -232,18 +232,106 @@ paths:
       tags:
       - Storage
       x-openapi-router-controller: schematic_api.controllers.storage_controller
-  /components/{componentLabel}/attributes:
+  /nodes/{nodeDisplay}/nodeLabel:
```

Comment:
Please refer to my comments in the function calls and apply changes to this file as well.